presto-main/src/main/java/com/facebook/presto/sql/analyzer/RelationId.java

```diff
@@ -40,6 +40,12 @@ public static RelationId anonymous()
         return new RelationId(null);
     }
 
+    @Nullable
+    public Node getSourceNode()
+    {
+        return sourceNode;
+    }
+
```

Comment:
it would be nice to separate this changes to different commit. Analyzer is completely different than engine security plugins.